tox.ini

```diff
@@ -1,15 +1,22 @@
 [tox]
-envlist = py27, py33, py34, py35, flake8
+envlist = {py27,py33,py34,py35}-{windows,linux,darwin},flake8
 skipsdist=True
 
 [testenv]
 usedevelop=True
 commands =
     py.test -v --cov=docker {posargs:tests/unit}
+platform =
+    windows: windows
+    linux: linux
+    darwin: darwin
+install_command = 
+    {py27,py33,py34,py35}-windows: pip install --only-binary=cryptography {opts} {packages}
```

Comment:
you probably want to put an install command here too, because otherwise things will break :(

```
{py27,...}-{darwin,linux}: pip install {opts} {packages}
```